app.cfg.example

```diff
@@ -356,12 +393,15 @@ scontrol_command = /usr/bin/scontrol
 #         are removed, the output (in PR comments) will lack important
 #         information.
 [submitted_job_comments]
-# awaits_release is no longer used since bot release v0.7.0
-# awaits_release = job id `{job_id}` awaits release by job manager
+awaits_release = job id `{job_id}` awaits release by job manager
```

Comment:
Why is the re-enabled? Based on that older comment, it shouldn't be used anymore?

Reply:
You're right, this branch probably forked before that change

Reply:
Fixed